prow/spyglass/viewers/buildlog/buildlog_template.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package buildlog
+
+const tmplt = `
```

Comment:
Replace constant templates like these with the compiled versions so that they are resolved at init time instead of when the view is actually used. Its safer and a bit more efficient. Example: https://github.com/kubernetes/test-infra/blob/5a33c36fa0440580f53d21b7aa3ba319e0a51aec/prow/cmd/deck/badge.go#L30-L51

Reply:
Interesting, I like it. Added in latest.